config/service_providers.yml

```diff
@@ -270,3 +270,20 @@ production:
     cert: 'identity_dashboard_cert'
     attribute_bundle:
       - email
+
+  # DHS / CBP / HRM
+  'urn:gov:gsa:SAML:2.0.profiles:sp:sso:DHS:CBP-HRM-DEV':
+    acs_url: 'https://uxvnwg001a5418.sat.cbp.dhs.gov:9002/hrm-services/saml/SSO'
+    assertion_consumer_logout_service_url: 'https://uxvnwg001a5418.sat.cbp.dhs.gov:9002/hrm-services/saml/logout'
+    sp_initiated_login_url: 'https://uxvnwg001a5418.sat.cbp.dhs.gov:9002/hrm-services/saml/login'
```

Comment:
@brbarbe2 Perfect Brad. 👌 That's the exact behavior we desire. We are not technically using this attribute today, but it has been part of UI flows we've implemented in the past and may again in the future. Instead of adding IdP-initiated authentication support we use this simple attribute to achieve the same functionality.